decrypt_test.go

```diff
@@ -59,3 +74,200 @@ o5FC5Jg2FUfFzZSDmyZ6IONUsdF/i78KDV5nRv1R+hI6/oRlWNCtTNBv/lvBBd6b
 dseUE9QoaQZsn5lpILEvmQJAZ0B+Or1rAYjnbjnUhdVZoy9kC4Zov+4UH3N/BtSy
 KJRWUR0wTWfZBPZ5hAYZjTBEAFULaYCXlQKsODSp0M1aQA==
 -----END PRIVATE KEY-----`
+
+//   - Generate new private key using openssl genrsa -out key.pem 2048
+//   - Create a self-signed certificate for key.pem using
+//     openssl req -x509 -key key.pem -out certificate.pem
+//   - Create a file with the data to be encrypted using
+//     echo -n "This is a test" > test.txt
+//   - Generate PKCS #7 enveloped data encrypted using AES using
+//     openssl smime -encrypt -in test.txt -aes256 -outform pem certificate.pem
+var RSAPKCS1v15EncryptedTestFixture = `
```

Comment:
the `embed` package could help you out here.

Reply:
This repo is still tested against Go 1.12, because it's a fork. I've added a note instead to look into that when the version is upped to at least 1.16. We're getting closer to actually maintaining the fork for others to be used, so after that we can think about upgrading Go versions, I would say.

Reply:
You could put these in files under `testdata` and load them during `TestMain` or `init` in the test file.

Reply:
It's a good idea, but I propose to keep it like this for a bit, as it's the format used by the upstream and I took their tests. There's some more cases not currently covered by tests, so I want to revisit this soon, but do it with `embed` at once.